### PR TITLE
Problem: no parsed log from old event

### PR DESCRIPTION
**Pull request by mmsqe**
👮🏻👮🏻👮🏻 !!!! REFERENCE THE PROBLEM YOUR ARE SOLVING IN THE PR TITLE AND DESCRIBE YOUR SOLUTION HERE !!!! DO NOT FORGET !!!! 👮🏻👮🏻👮🏻

# PR Checklist:

- [ ] Have you read the [CONTRIBUTING.md](https://github.com/crypto-org-chain/chain-main/blob/master/CONTRIBUTING.md)?
- [ ] Does your PR follow the [C4 patch requirements](https://rfc.zeromq.org/spec:42/C4/#23-patch-requirements)?
- [ ] Have you rebased your work on top of the latest master? 
- [ ] Have you checked your code compiles? (`make`)
- [ ] Have you included tests for any non-trivial functionality?
- [ ] Have you checked your code passes the unit tests? (`make test`)
- [ ] Have you checked your code formatting is correct? (`go fmt`)
- [ ] Have you checked your basic code style is fine? (`golangci-lint run`)
- [ ] If you added any dependencies, have you checked they do not contain any known vulnerabilities? (`go list -json -m all | nancy sleuth`)
- [ ] If your changes affect the client infrastructure, have you run the integration test?
- [ ] If your changes affect public APIs, does your PR follow the [C4 evolution of public contracts](https://rfc.zeromq.org/spec:42/C4/#26-evolution-of-public-contracts)?
- [ ] If your code changes public APIs, have you incremented the crate version numbers and documented your changes in the [CHANGELOG.md](https://github.com/crypto-org-chain/chain-main/blob/master/CHANGELOG.md)?
- [ ] If you are contributing for the first time, please read the agreement in [CONTRIBUTING.md](https://github.com/crypto-org-chain/chain-main/blob/master/CONTRIBUTING.md) now and add a comment to this pull request stating that your PR is in accordance with the [Developer's Certificate of Origin](https://github.com/crypto-org-chain/chain-main/blob/master/CONTRIBUTING.md#developer-certificate-of-originn).

Thank you for your code, it's appreciated! :)

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -239,7 +239,7 @@ replace (
 	// TODO: remove it: https://github.com/cosmos/cosmos-sdk/issues/13134
 	github.com/dgrijalva/jwt-go => github.com/golang-jwt/jwt/v4 v4.4.2
 	github.com/ethereum/go-ethereum => github.com/crypto-org-chain/go-ethereum v1.10.20-0.20231207063621-43cf32d91c3e
-	github.com/evmos/ethermint => github.com/crypto-org-chain/ethermint v0.6.1-0.20240419022701-412115d02b10
+	github.com/evmos/ethermint => github.com/crypto-org-chain/ethermint v0.6.1-0.20240430071430-1588ad231d11
```
Comment (mmsqe) [inline go.mod:242]: https://github.com/crypto-org-chain/ethermint/compare/412115d02b10...1588ad231d11

File: integration_tests/configs/upgrade-test-package.nix [hunk 2]
```diff
@@ -1,27 +1,23 @@
...
       outPath = builtins.fetchTarball "https://github.com/${repo}/archive/${rev}.tar.gz";
       inherit rev;
       shortRev = builtins.substring 0 7 rev;
     };
   }).defaultNix;
-  # v1.1.1
-  released = (fetchFlake "crypto-org-chain/cronos" "10b8eeb9052e3c52aa59dec15f5d3aca781d1271").default;
+  # v0.8.0
+  releasedGenesis = (fetchFlake "crypto-org-chain/cronos" "2f2cc88b501b47149690fdef05afbbbe5bc116c9").default;
+  # v1.0.15 a85daedc6ffc
```
Comment (yihuang) [inline integration_tests/configs/upgrade-test-package.nix:12]: comment don't match the actual commit hash?

File: integration_tests/cosmoscli.py [hunk 3]
```diff
@@ -746,19 +754,21 @@ def gov_propose_legacy(self, proposer, kind, proposal, mode="block", **kwargs):
...
+            "gov",
+            "vote",
+            proposal_id,
+            option,
+            "-y",
+            from_=voter,
+            home=self.data_dir,
+            **kwargs,
         )
+        # skip success log info that breaks json.loads
```
Comment (yihuang) [inline integration_tests/cosmoscli.py:768]: is it the keyring migration message?

Comment (mmsqe) [reply integration_tests/cosmoscli.py:768]: yea, separate migrate seems doesn't work

**Review by yihuang — APPROVED** [review]

**Review by trappeur21 — APPROVED** [review]

**Review by trappeur21 — APPROVED** [review]

**Pull request merged**